<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="PW_text">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">alternate-background-color: rgb(206, 92, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ID_text">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">alternate-background-color: rgb(206, 92, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check_btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>아이디 중복 확인</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name_text">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">alternate-background-color: rgb(206, 92, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signup_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>430</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>뒤로가기</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ID_text</tabstop>
  <tabstop>check_btn</tabstop>
  <tabstop>PW_text</tabstop>
  <tabstop>Name_text</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
